app/components/SideBar/index.js

```diff
@@ -19,7 +19,13 @@ class SideBar extends Component {
     this.interval = setInterval(() => this.props.updateBlockTimeSince(), 10000);
   }
 
+  onBottomSidebarClick() {
+    const a = document.getElementById("sidebar-menu-total-balance-extended");
```

Comment:
`getElementById` should not be used in a react application because of:
- strict selection
- breaking the reusability
- etc... 

It will be better to have a redux store, a state or a ref instead of this `getElementById` please